### PR TITLE
Docs: Change JS Foundation mentions to OpenJS Foundation

### PR DESCRIPTION
**Pull request by mgol**
### Summary ###

Docs: Update the copyright notice to mention the OpenJS Foundation

So far they were all about the JS Foundation.

### Checklist ###

* [x] All authors have signed the CLA at https://cla.js.foundation/jquery/jquery
* ~~New tests have been added to show the fix or feature works~~
* [x] Grunt build and unit tests pass locally with these changes
* ~~If needed, a docs issue/PR was created at https://github.com/jquery/api.jquery.com~~

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -1,3 +1,3 @@
-jQuery is a [JS Foundation](https://js.foundation/) project and subscribes to its code of conduct.
+jQuery is a [OpenJS Foundation](https://js.foundation/) project and subscribes to its code of conduct.
```
Comment (dmethvin) [inline CODE_OF_CONDUCT.md:1]: `an` instead of `a`

Comment (mgol) [reply CODE_OF_CONDUCT.md:1]: My find & replace was too quick!

Comment (mgol) [reply CODE_OF_CONDUCT.md:1]:
```suggestion
jQuery is an [OpenJS Foundation](https://js.foundation/) project and subscribes to its code of conduct.
```

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -1,3 +1,3 @@
-jQuery is a [JS Foundation](https://js.foundation/) project and subscribes to its code of conduct.
+jQuery is an [OpenJS Foundation](https://js.foundation/) project and subscribes to its code of conduct.
```
Comment (timmywil) [inline CODE_OF_CONDUCT.md:1]: The URL should also change to openjsf.org. It redirects there anyway.

Comment (mgol) [reply CODE_OF_CONDUCT.md:1]: Fixed. I also updated the Code of Conduct link.

Now, there's one remaining mention of js.foundation left, it's the CLA link: https://cla.js.foundation/jquery/jquery. I think that one needs to stay until we get switched to a different CLA checker.

Comment (timmywil) [reply CODE_OF_CONDUCT.md]: agreed

**Pull request merged**